opencga-app/src/main/resources/cli-usage.yml

```diff
@@ -0,0 +1,29 @@
+categories:
```

Comment:
An small description or commentary header about this new configuration file would be appreciated